### DISCUSSION TITLE
After upgrade from version 3 to 4, all CRLF changed to LF?

### DISCUSSION THREAD

**Discussion by jhgarrison** [joyfullservice/msaccess-vcs-addin#570] [Q&A]
As the title says, every file was touched, changing CRLF to LR.  Is this what is supposed to happen?  
hat this is
I also see other "interesting" changes, such as removal of "ColumnInfo" attributes from all forms. Just checking to verify this is expected.

**Comment by joyfullservice** [comment]
Moving from 3.x to 4.x is a pretty significant change, so yes, things like `ColumnInfo` being sanitized out are definitely expected. You will see a lot of different changes to source files after upgrading VCS due to changes and updates in version 4. As a general practice, I like to commit any work before upgrading VCS, then use a separate commit to reflect the changes due to the VCS upgrade.

I am not sure about the CRLF to LF changes... Could that be related to a setting in git? This add-in uses CRLF for line breaks.

**Reply by jhgarrison** [reply]
> As a general practice, I like to commit any work before upgrading VCS, then use a separate commit to reflect the changes due to the VCS upgrade

That's my plan :-)

> I am not sure about the CRLF to LF changes... Could that be related to a setting in git

Hmm... I don't think git is involved since all I've done is run the export against a database backed by a fully committed and pushed repo. I haven't issued any git commands yet.

**Reply by joyfullservice** [reply]
Regarding line endings, you might check out [this page](https://docs.github.com/en/get-started/getting-started-with-git/configuring-git-to-handle-line-endings), and review the default configuration in the [`.gitattributes` file](https://github.com/joyfullservice/msaccess-vcs-addin/blob/main/.gitattributes.default).
